packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1562,7 +1563,8 @@ export const DEFAULT_COMPONENT_ICONS: {
     ParticipantItemOptions: JSX.Element;
     ParticipantItemOptionsHovered: JSX.Element;
     ParticipantItemScreenShareStart: JSX.Element;
-    HoldCall: JSX.Element;
+    HoldCallContextMenuItem: JSX.Element;
```

Comment:
nit: Should be renamed to HoldCallContext**ual**MenuItem to be consistent with the other MenuItem names